tests/setupconfig/test_auth.py

```diff
@@ -0,0 +1,185 @@
+from django.core.management import CommandError, call_command
+
+import pytest
+import requests
+from django_setup_configuration.exceptions import ConfigurationRunFailed
+
+from mozilla_django_oidc_db.models import (
+    OpenIDConnectConfig,
+    UserInformationClaimsSources,
+)
+from mozilla_django_oidc_db.setupconfig.bootstrap import AdminOIDCConfigurationStep
+
+from ..conftest import KEYCLOAK_BASE_URL
+
+
+@pytest.mark.django_db
+def test_configure(setup_config_full):
+    call_command("setup_configuration")
```

Comment:
please provide `stdout` and `stderr` kwargs to suppress output in tests - slightly less relevant here because pytest captures the output by default, but in unittest based tests not doing this leads to so much clutter in the output. it's a good practice to always do this :)